lib/src/dartdoc.dart

```diff
@@ -144,19 +161,6 @@ class Dartdoc {
   @visibleForTesting
   set generator(Generator newGenerator) => _generator = newGenerator;
 
-  /// Factory method that builds Dartdoc with an empty generator.
-  factory Dartdoc.withEmptyGenerator(
```

Comment:
FYI, the `dart` tool is using this in its implementation of `dart doc`; see https://github.com/dart-lang/sdk/blob/main/pkg/dartdev/lib/src/commands/doc.dart#L119.

I'm sure its possible to update the tool for this change, but we'll want to do that as part of the next roll of this repo into the sdk.